includes/discovery/mempools/netspire.inc.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+if ($device['os'] === 'netspire') {
+    echo 'Netspire Memory Pool\n';
+    $usage = str_replace('"', "", snmp_get($device, 'netSpireDeviceStorageUsed.0', "-OQv", 'OACOMMON-MIB', 'openaccess'));
```

Comment:
You can drop the `openaccess` from the end of the `snmp_get()` call, this is already pulled in from the yaml file.